src/constants.h

```diff
@@ -52,3 +52,9 @@
  * Any other means that index-1 is the index of token on the token array
  */
 #define TOKEN_DATA_INDEX_MASK 0x7Fu
+
+/**
+ * Maximum length of a data script
+ * 150 max data + 2 opcodes + 1 byte of length (OP_PUSHDATA1 length ...data OP_CHECKSIG)
+ */
+#define MAX_DATA_SCRIPT_LEN 153
```

Comment:
The 150 max length is not defined in the protocol, but it is the max length of data allowed when creating a token with the desktop wallet.
The fullnode allows up to 1Kb (for validation purposes) but our public nodes only allow txs to be pushed via api with up to 255 byte scripts.

Since the device will be used with the desktop wallet, i will use the desktop wallet limit which is the lowest.